src/tbbmalloc/lin64-tbbmalloc-export.def

```diff
@@ -50,9 +50,182 @@ _ZN3rml10pool_msizeEPNS_10MemoryPoolEPv;
 local:
 
 /* TBB symbols */
-*3rml*;
+_Z28doThreadShutdownNotificationPN3rml8internal7TLSDataEb;
+_ZN3rml8internal10BackRefIdx10newBackRefEb;
```

Comment:
Why are you exporting internal mangled symbols?

Reply:
These are in the local part of the linker script. I'm not exporting anything that TBB doesn't itself export, just replacing the usage of wildcards as lld doesn't prioritize non-wildcards over wildcards as ld does.